### ISSUE TITLE
AlertDialog icon is cutted

### ISSUE THREAD

**Issue by BraisGabin** [Prototik/HoloEverywhere#732]
I think this is a problem with the AlertDialog's title height. You can see this issue at the demo app (6a271dbc1c2abc924ffc324d293cd8f812586a38).

AlertDialog's title height OK:
![device-2013-12-19-140827](https://f.cloud.github.com/assets/721244/1782510/174dc43e-68af-11e3-9be5-abe796c63622.png)
AlertDialog's title height KO:
![device-2013-12-19-140846](https://f.cloud.github.com/assets/721244/1782511/174f718a-68af-11e3-8af2-9623f54c29fb.png)

I test this problem with v4 and it works fine.

Nexus 5 (4.4.2)

**Comment by httpdispatch** [comment]
@BraisGabin did you find any solution for this issue?

**Comment by httpdispatch** [comment]
Looks like it is related to this http://stackoverflow.com/questions/20121711/dialog-on-android-kitkat-seems-to-be-cut

**Comment by httpdispatch** [comment]
Looks like this solution works http://stackoverflow.com/a/20782404/527759

**Comment by BraisGabin** [comment]
No, I found this bug just reviewing the sample app. I didn't need a work around in my code.

**Comment by httpdispatch** [comment]
Perhaps somebody else will face the same issue so would be good if they'll find the solution here
